### PR TITLE
doc: Add TOC for each doc parent page

### PR DESCRIPTION
**Pull request by mantissahz**
longhorn/longhorn#4754

### CODE REVIEW

**Review by c3y1huang — APPROVED** [review]
LGTM.

I assume there is an agreement not to fix the latest subversion of the previous major versions (1.3.3, 1.2.7). /cc @innobead

File: content/docs/1.4.0/advanced-resources/_index.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
...
+* [Orphaned Data Cleanup](./orphaned-data-cleanup)
+* [ReadWriteMany (RWX) Volume](./rwx-workloads)
+* [Restore cluster with a Rancher snapshot](./rancher-cluster-restore)
+* [Snapshot Data Integrity Check](./snapshot-data-integrity-check)
+* [Use Longhorn Volume as an iSCSI Target](./iscsi)
+* [Migrating from the Flexvolume Driver to CSI](./migrating-flexvolume)
+* [Configuring Defaults for Nodes and Disks](./default-disk-and-node-config)
+* [Troubleshooting](./troubleshooting)
+* [Support Bundle](./support-bundle)
+* [Deploy](./deploy/)
```
Comment (c3y1huang) [inline content/docs/1.4.0/advanced-resources/_index.md:17]: nit: do we need the trailing slash? This seems a bit inconsistent.

Comment (mantissahz) [reply content/docs/1.4.0/advanced-resources/_index.md:17]: Just tell it contains sub pages, but it could be removed.
Will do.

**Review by c3y1huang — APPROVED** [review]

File: content/docs/1.4.0/advanced-resources/_index.md (hunk 1 above)
Comment (mantissahz) [reply content/docs/1.4.0/advanced-resources/_index.md]: Removed. Thanks

**Pull request merged**